src/services/usersService.ts

```diff
@@ -56,6 +56,20 @@ export class UsersService {
         }
     }
 
+    public getTokenFromTiketParams(parameters: URLSearchParams): string {
```

Comment:
Typo in method name, 'Ticket' not 'Tiket'

Reply:
Thank you! good catch :) will fix it